code/modules/projectiles/guns/ballistic/pistol.dm

```diff
@@ -80,3 +80,16 @@
 		to_chat(user, "<span class='notice'>..and falls into view. Whew, that was a close one.</span>")
 		user.dropItemToGround(src)
 
+/obj/item/gun/ballistic/automatic/pistol/lathe
+	name = "lathe pistol"
+	desc = "A makeshift pistol with an internal microlathe which prints bullets out of a magnetized alloy of metal and glass which are then magnetically shot through a magnetic coil. Has a short delay so the lathe can print another round."
```

Comment:
desc = "A makeshift pistol with an internal microlathe. Has a short firing delay and requires a glass metal alloy round to create bullets." (or something similar, your description is far too long)